custom/src/main/java/com/solarwinds/opentelemetry/extensions/initialize/ConfigurationLoader.java

```diff
@@ -598,4 +637,9 @@ public static boolean shouldUseOtlpForMetrics() {
     return (enabled == null || enabled)
         && (collectorEndpoint == null || !collectorEndpoint.contains("appoptics.com"));
   }
+
+  public static boolean shouldUseOtlpForTraces() {
+    String collectorEndpoint = (String) ConfigManager.getConfig(ConfigProperty.AGENT_COLLECTOR);
+    return (collectorEndpoint == null || !collectorEndpoint.contains("appoptics.com"));
+  }
```

Comment:
Is this method ever used outside of tests ?

Reply:
It doesn't look like it is.